packages/medusa/src/api/routes/admin/draft-orders/create-draft-order.ts

```diff
@@ -34,7 +34,6 @@ import { IsType } from "../../../../utils/validators/is-type"
  *     application/json:
```

Comment:
Question: would it required some other changes around the service layer as well?